optimizex.io.connect.json

```diff
@@ -0,0 +1,19 @@
+{
+  "providerId": "optimizex.io",
+  "providerName": "Optimizex",
+  "serviceId": "ox-connect",
+  "serviceName": "Optimizex Public Web Application",
+  "version": 1,
+  "logoUrl": "",
+  "description": "Enables a Domain to work with Optimizex",
+  "variableDescription": "HOST is the subdomain for the domain.",
+  "hostRequired": true,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%ox-cname-host%",
```

Comment:
if the subdomain is set with `host` parameter, then here I would expect `@` not a variable. Note that `@` will be replaced by the host to render the full hostname to write into the zone.

Reply:
Hi @pawel-kow
We need to set up this variable, individually for our clients, for example: for one client good is ox.site1.com, and for another mysybdomain.site2.com

Reply:
understood, in this case `@` is the right `host` value for the template.
In the application the `host` parameter will be used in place of `@`.

So, the examples would be called with `domain=site1.com` and `host=ox` or `domain=site2.com` and `host=mysubdomain` respectively.

Details on this mechanism in the specificaition https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#hostname-in-template